verticles/graphql/src/main/java/com/gentics/mesh/graphql/filter/CommonFields.java

```diff
@@ -0,0 +1,44 @@
+package com.gentics.mesh.graphql.filter;
+
+import com.gentics.graphqlfilter.filter.DateFilter;
+import com.gentics.graphqlfilter.filter.FilterField;
+import com.gentics.graphqlfilter.filter.MappedFilter;
+import com.gentics.graphqlfilter.filter.StringFilter;
+import com.gentics.mesh.core.data.NamedElement;
+import com.gentics.mesh.core.data.UserTrackingVertex;
+import com.gentics.mesh.graphdb.model.MeshElement;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Common fields that can be composed in various types
+ */
+public final class CommonFields {
+	private CommonFields() {}
+
+	public static <T extends MeshElement> FilterField<T, Map<String, ?>> uuidFilter() {
+		return new MappedFilter<>("uuid", "Filters by uuid", StringFilter.filter(), MeshElement::getUuid);
+	}
+
+	public static <T extends NamedElement> List<FilterField<T, Map<String, ?>>> nameFilter() {
+		return Arrays.asList(
+			uuidFilter(),
```

Comment:
Remove uuid filter here. It is better to let the implementer choose which filters to include.